posts/2024-03-08-Rustup-1.27.0.md

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
Have we mentioned new maintainers of Rustup anywhere yet? It would be great if we can give credits to past and new members :)

Reply:
Done!